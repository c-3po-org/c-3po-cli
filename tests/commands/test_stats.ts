import * as path from "path";
import { execSync } from "child_process";

const poPath = path.resolve(__dirname, "../fixtures/statsTest/stats.po");

test("display basic file stats", () => {
    const result = execSync(`ts-node src/index.ts stats ${poPath}`).toString();
    expect(result).toMatchSnapshot();
});
