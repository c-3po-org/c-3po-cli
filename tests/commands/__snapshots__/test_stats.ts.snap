// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display basic file stats 1`] = `
"TOTAL: 3
CONTEXTS: 1
TRANSLATED: 1
FUZZY: 0
[#################·································] 33% 1/2/3
"
`;
